packages/server/lib/controllers/proxy.controller.ts

```diff
@@ -368,10 +367,10 @@ class ProxyController {
 
     private async reportError(error: AxiosError, url: string, config: ApplicationConstructedProxyConfiguration, errorMessage: string, logCtx: LogContext) {
         const safeHeaders = proxyService.stripSensitiveHeaders(config.headers, config);
-        await logCtx.error('The provider responded back with an error code', {
+        await logCtx.error(`${error.request?.method.toUpperCase()} ${url} failed with status '${error.response?.status}'`, {
             code: error.response?.status,
             url,
-            error: errorMessage,
+            error: new Error(errorMessage),
```

Comment:
must be an error otherwise show as "unknown error" https://github.com/NangoHQ/nango/blob/8bcd76bab6181541c03feb77b622b7c38d1a818a/packages/logs/lib/client.ts#L65

Reply:
good catch